port/ztpf/omrsignal.c

```diff
@@ -1321,12 +1321,17 @@ sig_full_shutdown(struct OMRPortLibrary *portLibrary)
 		}
 		omrthread_monitor_exit(asyncReporterShutdownMonitor);
 #endif	/* defined(OMR_PORT_ASYNC_HANDLER) */
-		omrthread_monitor_enter(ztpfSigShutdownMonitor);
-		shutdown_ztpf_sigpush = 1;
-		while (shutdown_ztpf_sigpush) {
+                // Assume if zTPF signal monitor is gone no need to shut it down.
+                if (ztpfSigShutdownMonitor != 0)
+		{
+		   omrthread_monitor_enter(ztpfSigShutdownMonitor);
+		   shutdown_ztpf_sigpush = 1;
+		   while (shutdown_ztpf_sigpush)
+		   {
```

Comment:
Applied coding standards in the below suggestion: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md.
```suggestion
                /* Assume if zTPF signal monitor is gone no need to shut it down. */
                if (NULL != ztpfSigShutdownMonitor) {
			omrthread_monitor_enter(ztpfSigShutdownMonitor);
			shutdown_ztpf_sigpush = 1;
			while (0 != shutdown_ztpf_sigpush) {
				omrthread_monitor_wait(ztpfSigShutdownMonitor);
			}
			omrthread_monitor_exit(ztpfSigShutdownMonitor);
		}
```